executor/executor.go

```diff
@@ -2297,6 +2294,12 @@ func ResetContextOfStmt(ctx sessionctx.Context, s ast.StmtNode) (err error) {
 	vars.DurationWaitTS = 0
 	vars.CurrInsertBatchExtraCols = nil
 	vars.CurrInsertValues = chunk.Row{}
+
+	// FIXME: perform plan digest generation only when necessary
+	if variable.EnableResourceControl.Load() {
+		_, planDigest := getPlanDigest(sc)
```

Comment:
There is no `Plan` in `sc` here, so can not get plan. digest

Reply:
I see